src/app/lib/views/browser/list.js

```diff
@@ -273,7 +280,7 @@
                     '</div>' +
                 '</div>';
 
-            var ghosts = '<div class="ghost"></div>'.repeat(10);
+            var ghosts = '<div class="ghost"></div>'.repeat(this.itemsPerRow()*2);
```

Comment:
we're limited in our postersize, so *2 is a safe bet. I'll check if I can do something with minPerRow, it might be better